tests/test_upload.py

```diff
@@ -72,6 +73,31 @@ def test_successs_prints_release_urls(upload_settings, stub_repository, capsys):
     assert captured.out.count(NEW_RELEASE_URL) == 1
 
 
+def test_successs_prints_uploaded_package_names_if_verbose(upload_settings, capsys):
+    """Print the path and file size of each distribution attempting to be uploaded."""
```

Comment:
Repeating an [earlier comment](https://github.com/pypa/twine/pull/652#discussion_r437299337): I think think is more accurate/descriptive, and follows are recent agreement to use imperative mood in docstrings:

```suggestion
def test_print_packages_if_verbose(upload_settings, capsys):
    """Print the path and file size of each distribution attempting to be uploaded."""
```